.github/workflows/ci.yml

```diff
@@ -0,0 +1,72 @@
+name: ci
+
+on: pull_request
```

Comment:
and also on branches?

Reply:
https://github.com/elastic/synthetics/pull/728/commits/13fc64e9c21bef3116416e7d8aefd0fb44887f14